components/brave_wallet/browser/solana_provider_impl.h

```diff
@@ -78,23 +79,30 @@ class SolanaProviderImpl final : public mojom::SolanaProvider,
                                      const std::string& account,
                                      SignMessageCallback callback,
                                      bool approved,
-                                     const std::string& signature,
-                                     const std::string& error);
-  void OnSignTransactionRequestProcessed(std::unique_ptr<SolanaTransaction> tx,
-                                         SignTransactionCallback callback,
-                                         bool approved);
+                                     mojom::ByteArrayStringUnionPtr signature,
+                                     const absl::optional<std::string>& error);
+  void OnSignTransactionRequestProcessed(
+      std::unique_ptr<SolanaTransaction> tx,
+      const std::string& account,
+      SignTransactionCallback callback,
+      bool approved,
+      mojom::ByteArrayStringUnionPtr signature,
+      const absl::optional<std::string>& error);
   void OnSignAllTransactionsRequestProcessed(
       const std::vector<std::unique_ptr<SolanaTransaction>>& txs,
+      const std::string& account,
       SignAllTransactionsCallback callback,
-      bool approved);
+      bool approved,
+      absl::optional<std::vector<mojom::ByteArrayStringUnionPtr>> signatures,
+      const absl::optional<std::string>& error);
   void OnAddUnapprovedTransaction(SignAndSendTransactionCallback callback,
                                   bool success,
                                   const std::string& tx_meta_id,
                                   const std::string& error_message);
 
-  absl::optional<SolanaMessage> GetDeserializedMessage(
-      const std::string& encoded_serialized_msg,
-      const std::string& account);
+  absl::optional<std::pair<SolanaMessage, std::vector<uint8_t>>>
```

Comment:
can we add a comment saying second of this pair is?

Reply:
https://github.com/brave/brave-core/pull/14099/commits/2ccc882ec6bdd8602308599d3048016df440e19b